util/K8sUtil.go

```diff
@@ -5,9 +5,12 @@ import (
 	"fmt"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/validation"
+	"regexp"
 	"strings"
 )
 
+const EphemeralServerVersionRegex = "v[1-9]\\.[2-9][3-9].+"
```

Comment:
why haven't we used ``v[1-9]\\.[2-9][3-9].*`` ??

Reply:
we should use this ,since it matches with zero or more chars